packages/mds-db/tests/mds-db.spec.ts

```diff
@@ -412,6 +412,10 @@ if (pg_info.database) {
 
       it('can edit a Geography', async () => {
         const geography_json = clone(DistrictSeven.geography_json)
+        assert.notEqual(geography_json, null)
+        if (geography_json == null) {
```

Comment:
It should never be null the way this test is currently written, but because `geography_json` is now an optional property ([refs](https://github.com/lacuna-tech/mds-core/pull/10#discussion_r334621174)) this check is needed to make the compiler happy.